types/staking.go

```diff
@@ -8,11 +8,11 @@ const (
 
 	// default maximal voting power ratio
 	// Max percentage of the total voting power a single validator can have
-	DefaultMaxVotingPowerRatio = "0.1"
+	DefaultMaxVotingPowerRatio = "0.2"
```

Comment:
10% is hit pretty easily when there's little validators, set to 20% and allow the param to be tuned in the future